examples/tree_2d_dgsem/elixir_advection_amr_coarsen_twice.jl

```diff
@@ -25,10 +25,11 @@ end
 function (indicator::IndicatorAlwaysCoarsen)(u::AbstractArray{<:Any, 4},
                                              mesh, equations, dg, cache;
                                              t, kwargs...)
+    RealT = eltype(u)
     alpha = indicator.cache.alpha
     resize!(alpha, nelements(dg, cache))
 
-    alpha .= -1.0
+    alpha .= RealT(-1)
```

Comment:
```suggestion
    fill!(alpha, -1)
```

Since we just fill an array, we can convert the integer to the element type automatically.